tests/basic.json

```diff
@@ -1,6 +1,11 @@
 {
   "title": "basic",
   "description": "basic view definition",
+  "fhirVersion": [
+    "5.0",
+    "4.0",
+    "3.0"
+  ],
```

Comment:
I think these are not actual FHIR versions - they should correlate with one of the versions in the table at the top of this page: https://hl7.org/fhir/history.html

Reply:
I think what you have done is reasonable, and it is good that this lines up with the FHIR version binding in the ViewDefinition resource.

I think we should probably add the clarification that they should be release versions, i.e. `3.0.2`, `4.0.1` and `5.0.0`.

Reply:
Okay I changed these to 3 segments versions, e.g., `5.0.0` and `4.0.1`.

Reply:
@johngrimes if you have any other comments, please let me know. Otherwise, I will merge this PR today.